pkg/common/nats/conn.go

```diff
@@ -82,6 +87,9 @@ func NewNatsConn(ctx context.Context, config commonconfig.EventingNatsConfig) (*
 	// reconnection options
 	if config.ConnOpts != nil && config.ConnOpts.RetryOnFailedConnect {
 		reconnectWait := time.Duration(config.ConnOpts.ReconnectWaitMilliseconds) * time.Millisecond
+		reconnectJitter := defaultJitterIfEmpty(config.ConnOpts.ReconnectJitterMilliseconds, defaultReconnectJitter)
+		reconnectJitterTLS := defaultJitterIfEmpty(config.ConnOpts.ReconnectJitterMilliseconds, defaultReconnectJitterTLS)
```

Comment:
typo on the `ConnOpts` field...

```suggestion
		reconnectJitterTLS := defaultJitterIfEmpty(config.ConnOpts.ReconnectJitterTLSMilliseconds, defaultReconnectJitterTLS)
```